From 6e429abdf7dddad875318ec4311f51607b6c8391 Mon Sep 17 00:00:00 2001
From: Chingling Wang <chingling.wang@nxp.com>
Date: Wed, 11 Mar 2026 10:03:47 -0700
Subject: [PATCH] YOCIMX-9796 imx-gpu-sdk build break on wrynose

Replace fmt::localtime with std::localtime since fmt::localtime is decrecated

Signed-off-by: Chingling Wang<chingling.wang@nxp.com>
---
 .../source/Shared/UI/Benchmark/Scene/ResultScene.cpp          | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/DemoApps/Shared/UI/Benchmark/source/Shared/UI/Benchmark/Scene/ResultScene.cpp b/DemoApps/Shared/UI/Benchmark/source/Shared/UI/Benchmark/Scene/ResultScene.cpp
index d43482a86..de02fbc6c 100644
--- a/DemoApps/Shared/UI/Benchmark/source/Shared/UI/Benchmark/Scene/ResultScene.cpp
+++ b/DemoApps/Shared/UI/Benchmark/source/Shared/UI/Benchmark/Scene/ResultScene.cpp
@@ -58,6 +58,8 @@
 #include "Control/CustomControlFactory.hpp"
 #include "Control/CustomUIConfig.hpp"
 #include "SceneCreateInfo.hpp"
+#include <chrono>
+#include <ctime>
 
 namespace Fsl
 {
@@ -230,7 +232,7 @@ namespace Fsl
       // layout->AddChild(labelVal4, 1, 4);
       // return layout;
 
-      auto dateString = fmt::format("{0:%Y-%m-%d} {0:%H:%M:%S}", fmt::localtime(sourceData.Time));
+      auto dateString = fmt::format("{0:%Y-%m-%d} {0:%H:%M:%S}", *(std::localtime(&sourceData.Time)));
       auto debugString = sourceData.Info.AppDebugMode ? StringViewLite(" (Debug)") : StringViewLite();
       auto descLabel =
         uiFactory.CreateLabel(fmt::format(FMT_STRING("Scene: {}, {}x{}px {}dpi, {}, {} V{}{}"), static_cast<uint32_t>(sourceData.Info.Scene),
-- 
2.43.0

